deps/prebuilt/include/TAEF/WexTestClass.h

```diff
@@ -115,13 +119,44 @@ extern "C" __declspec(noreturn) void __fastfail(unsigned int);
 #define TAEF_FAST_FAIL System::Environment::FailFast(nullptr)
 #endif // #if !defined(__cplusplus_cli)
 
+#if defined(__clang__)
```

Comment:
Can you move this #include further up? Even though it's clang-specific it should be with the rest (somewhere near the 'except.h' include).